src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -1,39 +1,48 @@
 import {observer} from 'mobx-react-lite';
-import React, {useContext} from 'react';
-import {Button, ScrollView, StyleSheet, Text} from 'react-native';
-import {PrimaryFonts, SecondaryFonts} from '../../assets';
+import React, {useContext, useEffect} from 'react';
+import {Button, ScrollView, StyleSheet, Text, View} from 'react-native';
+import {SecondaryFonts} from '../../assets';
 import {RootStoreContext} from '../../models';
 import {HomeScreenProps} from './HomeScreen.types';
+import {BusinessServiceTypes} from '../../services';
 import {Strings} from '../../i18n';
 
 const HomeScreen = observer((_props: HomeScreenProps) => {
-  const {count, increamentCount, Count, decrementCount} =
-    useContext(RootStoreContext);
+  const {businesses} = useContext(RootStoreContext);
+
+  useEffect(() => {
+    businesses.getAllBusinesses(
+      'Indiana 1209jaoineoiance',
+      false,
+      BusinessServiceTypes.SearchBusinessesSortBy.Best_Match,
+      10,
+    );
+  }, [businesses]);
```

Comment:
Why businesses? Wont this call the API continuously? because every time the model changes, you are hitting the API

Reply:
I missed it and i fixed it now.